binr/r2r/meson.build

```diff
@@ -3,8 +3,8 @@ if get_option('enable_r2r')
   executable('r2r', ['r2r.c', 'load.c', 'run.c'],
     include_directories: [platform_inc],
     dependencies: [
-      r_util_dep,
-      r_cons_dep,
+      r_util_static_dep,
```

Comment:
im "fine", but imho dynamic linking should be fine if you have your r2 properly build and properly installed in your system. see env.sh to setup the LD_LIBRARY_PATH of use the rpath LDFLAGS

Reply:
It is still same code (just linked statically). I'd be more afraid that different code can sneak in if `r2r` loads wrong dynamic library (like one from system-installed radare2).